doc/_static/custom_styles.css

```
@@ -52,3 +52,44 @@ div.fa-globe-americas::before {
 div.fa-user-friends::before {
   color: #f6951f;
 }
+
+/*** Styling for quickstart interactive tour ***/
+
+.interactive-tour-container {
+  position: relative;
+  display: inline-block;
+  transition: transform 500ms ease-in-out;
+}
+
+.interactive-tour-container .tour-screenshot {
+   display: none;
+   max-width: 100%;
+   height: auto;
```

```suggestion
  display: none;
  max-width: 100%;
  height: auto;
```

Extra space on the indent, my mistake